flink-annotations/src/main/java/org/apache/flink/annotation/docs/Documentation.java

```diff
@@ -91,6 +91,7 @@
 		public static final String EXPERT_ZOOKEEPER_HIGH_AVAILABILITY = "expert_high_availability_zk";
 		public static final String EXPERT_SECURITY_SSL = "expert_security_ssl";
 		public static final String EXPERT_ROCKSDB = "expert_rocksdb";
+		public static final String EXPERT_CLUSTER = "expert_cluster";
```

Comment:
You need to include this section into the `config.md` for it to show up.

Reply:
Thanks. Will do!